File: string/fizz-buzz/fizz-buzz.ts
function fizzBuzz(n: number): string[] {
    const answer = Array<string>(n);

    for (let i = 0; i < n; i++) {
        const num = i + 1;
        let ans = "";
        if (num % 3 === 0) {
            ans += "Fizz";
        }
        if (num % 5 === 0) {
            ans += "Buzz";
        }
        if (ans === "") {
            ans = `${num}`;
        }
        answer[i] = ans;
    }

    return answer;
};

File: string/fizz-buzz/fizz-buzz.go
package fizzbuzz

import "strconv"

func fizzBuzz(n int) []string {
	answer := make([]string, n)
	for i := 0; i < n; i++ {
		c := i + 1
		ans := ""
		if c%3 == 0 {
			ans += "Fizz"
		}
		if c%5 == 0 {
			ans += "Buzz"
		}
		if ans == "" {
			ans = strconv.Itoa(c)
		}
		answer[i] = ans
	}

	return answer
}
